Tests/LibWeb/Text/input/HTML/HTMLSelectElement-value-change-trigger-onchange.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```

Comment:
Commit title: please use assertive language about what is being changed. Also, you're changing `LibWeb`, not `LibJS`.

Example: "LibWeb: Trigger `onchange` event when setting \<select\> value"

Reply:
Ah I got that backwards :)